media-fonts/ricty/Manifest

```diff
@@ -1 +1,3 @@
 DIST Ricty-3.2.0.tar.gz 13286 SHA256 bb2bf6d7762b5e6e13d237bd82ba43d6faffc17b7e6d541db89d58a73a8a319e
+DIST Ricty-3.2.0.tar.gz 13286 SHA256 bb2bf6d7762b5e6e13d237bd82ba43d6faffc17b7e6d541db89d58a73a8a319e SHA512 77b7416e8c2e6847ec4c964dda8e4fa52ba3e63751ce030fbd0cd83a07a88c777319690777e48e945d36ebf06fe4717d10a7e02193f48575507c5c1f71446b36 WHIRLPOOL cc86cd4da191941ea13bf3900ff78e2b0cd2c4825e79d23b0b00543ca3180fbc8afdc0fc94876d0ed89cc7de0c9ff8332b4877551f06f98a70cd5a96fa2aef47
+EBUILD ricty-3.2.0-r1.ebuild 1065 SHA256 c1ea98f965c9a016475558b70168d73582e112cf10888585161726a5a68d8523 SHA512 6186b5abe1de7202070023475f28ac5bc66d224a4353a700af31a29ec117d5c606bf14993b4e2185e68eccf162f820470d8df08ae2425463967fd27000c0fb98 WHIRLPOOL 09c0bcb3e9a7954954072bce75b90117dd9ebad4a6189695850110cfe106b2c73e846da380cd428331a9e34275d57435cd5b91e3d1482b9cb1feb70edc0a96ff
```

Comment:
this shoudln't be here... please use repoman on the real tree. Ebuild checksums are deprecated.